torch/nn/functional.py

```diff
@@ -613,15 +625,35 @@ def alpha_dropout(input, p=0.5, training=False, inplace=False):
     return f(input, p, training)
 
 
-def dropout2d(input, p=0.5, training=False, inplace=False):
+def dropout2d(input, p=0.5, training=True, inplace=False):
+    r"""
+    Randomly zeroes whole channels (a channel a 2D slice of dimensions H, W) of the input tensor.
+
+    See :class:`~torch.nn.Dropout2d` for details.
+
+    Args:
+        p: probability of an element to be zeroed. Default: 0.5
+        training: apply dropout if is True. Defualt: True
+        inplace: If set to ``True``, will do this operation in-place. Default: ``False``
+    """
     if p < 0 or p > 1:
         raise ValueError("dropout probability has to be between 0 and 1, "
                          "but got {}".format(p))
     f = _VF.feature_dropout_ if inplace else _VF.feature_dropout
     return f(input, p, training)
 
 
-def dropout3d(input, p=0.5, training=False, inplace=False):
+def dropout3d(input, p=0.5, training=True, inplace=False):
+    r"""
+    Randomly zeroes whole channels (a channel is a 3D slice of dimensions D, H, W) of the input tensor.
+
+    See :class:`~torch.nn.Dropout3d` for details.
+
+    Args:
+        p: probability of an element to be zeroed. Default: 0.5
+        training: apply dropout if is True. Defualt: True
```

Comment:
nit: format these booleans in all the doc strings like you did for inplace

Reply:
right, I will change it